src/systemTest/java/com/flightstats/hub/system/service/ChannelItemPathPartExtractor.java

```diff
@@ -0,0 +1,35 @@
+package com.flightstats.hub.system.service;
+
+import com.flightstats.hub.util.TimeUtil;
+import org.joda.time.DateTime;
+
+public class ChannelItemPathPartExtractor {
```

Comment:
I was kind of unhappy with the way I split out responsibilities on this + the builder + the lombok object. I like the idea of overriding the builder, though. I had originally gotten rid of the lombok object entirely and had this as its replacement. I added it so that we didn't do a lot of recalculations, but that's kind of a silly consideration for a test class. I'll rework this a little and see what you think.